addon/helpers/choice.js

```diff
@@ -9,6 +9,11 @@ export default MicroState.extend({
       return Type.create(values, options);
     },
 
+    set(choice, values, options) {
```

Comment:
@cowboyd can you give me an example of a JSON structure what could be passed into set? I'm just trying to figure out what should test for, because I'm not clear at the moment.